drivers/iio/frequency/ad917x/ad917x_jesd_api.c

```diff
@@ -243,6 +243,52 @@ int ad917x_jesd_get_cfg_param(ad917x_handle_t *h,
 	return API_ERROR_OK;
 }
 
+int ad917x_jesd_oneshot_sync(ad917x_handle_t *h, u8 err_window)
+{
+	int err;
+	u8 mode;
+
+	if (h == NULL)
+		return API_ERROR_INVALID_HANDLE_PTR;
+
+	err = ad917x_register_write(h, AD917X_SYSREF_ERR_WINDOW_REG,
+				    AD917X_SYSREF_ERR_WIN(err_window));
+	if (err != API_ERROR_OK)
+		return err;
+
+	err = ad917x_register_read(h, AD917X_SYSREF_MODE_REG, &mode);
+	if (err != API_ERROR_OK)
+		return err;
+
+	if (mode & AD917X_SYSREF_MODE_ONESHOT) {
+		err = ad917x_register_write(h, AD917X_SYSREF_MODE_REG,
+					    mode & ~AD917X_SYSREF_MODE_ONESHOT);
+		if (err != API_ERROR_OK)
+			return err;
+	}
+
+	err = ad917x_register_write(h, AD917X_SYSREF_MODE_REG,
+				    mode | AD917X_SYSREF_MODE_ONESHOT);
+	return err;
+}
+
+int ad917x_jesd_get_sync_rotation_done(ad917x_handle_t *h, bool *done)
+{
+	int err;
+	u8 mode;
+
+	if (h == NULL)
+		return API_ERROR_INVALID_HANDLE_PTR;
+
+	err = ad917x_register_read(h, AD917X_SYSREF_MODE_REG, &mode);
+	if (err != API_ERROR_OK)
+		return err;
+
+	*done = (mode & AD917X_SYNC_ROTATION_DONE);
+
+	return API_ERROR_OK;
```

Comment:
```suggestion
	return 0;
```

We avoid the api defines at the linux driver layer.

Reply:
Now returning 0.